CONTRIBUTING.md

```diff
@@ -130,6 +131,26 @@ when creating documentation. Refer to our [Component Guide](COMPONENT_GUIDE.md)
 4. If you are willing to assist in the process of creating a new page, then follow the instructions above to `Fork` or `Clone` the repo and make your edits.
 5. Follow the instructions above to submit a `PR` for your change.
 
+### Adding a "what's new" post
+
+We use "What's new" posts, or NR1 announcements, to let people know about new features and updates in [New Relic One](https://one.newrelic.com/launcher/jerome.plg-whats-new-launcher?pane=eyJuZXJkbGV0SWQiOiJqZXJvbWUucGxnLXdoYXRzLW5ldyJ9). You can also find What's new on our [Docs site](https://docs.newrelic.com/whats-new). To add a new post:
+
+1. If it doesn't exist, create the folder for the current year and month in the [`src/content/whats-new/`](https://github.com/newrelic/docs-website/tree/develop/src/content/whats-new) directory (i.e. `src/content/whats-new/2020/12` for December 2020).
+2. Create a markdown file (`.md`) in the right year and month directory (**note:** this is _not_ a MDX file).
+3. Add this to the top of your markdown file. This is the frontmatter for What's new posts.
+
+| field            | Description
+| ---------------- | -----------
+| `contentType`    | This value should _always_ be `nr1Announcement` for what's new posts
+| `template`       | This value should _always_ be `whatsNew` for what's new posts
+| `title`          | The title of the post
+| `summary`        | A short summary of the post (shown [within the product](https://one.newrelic.com/launcher/jerome.plg-whats-new-launcher?pane=eyJuZXJkbGV0SWQiOiJqZXJvbWUucGxnLXdoYXRzLW5ldyJ9) and the [documentation site](https://docs.newrelic.com/whats-new))
+| `releaseDate`    | The date of the post in the format `YYYY-MM-DD` (i.e. `2020-12-05` For December 5, 2020)
+| `learnMoreLink`  | A link to learn more about the feature being announced (i.e. a link to the blog)
+| `getStartedLink` | A link to a relevant documentation site post
+
+4. Submit a pull request for your new what's new post.
```

Comment:
I think we need to be consistent with our capitalization.

"What's new" does feel a little weird, but I worry that "what's new" just blends in.